posts/2023-02-10-transition-to-devel/index.qmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Renaming the Default Branch to Devel"
+description: |
+  During the 3.17 devel cycle, Bioconductor will rename the default branch devel.
+author:
+  - name: J Wokaty
+    url: https://github.com/jwokaty
+date: 2023-02-10
+categories: BioC
+---
+
+In the next few weeks, Bioconductor will rename the default branch on
+`git.bioconductor.org` to `devel`. This change has been many discussions and a
+lot of work in the making. Here's why we're doing it and how it impacts your
+work.
+
+## Moving Toward Inclusion
+
+You might already be familiar with other organizations moving away from the
+terminology "master" to use more inclusive language. We realize that this
+language is not representative of Bioconductor and it's a step toward inclusion
+that's inline with Bioconductor's
+[Code of Conduct](https://www.bioconductor.org/about/code-of-conduct/)
+to provide a welcoming and supportive experience.
+
+We specifically chose `devel` as it more aptly describes what this branch is
+within the context of Bioconductor.
+
+## Impact on Bioconductor Package Developer Work
+
+After TBD, developers will need to push their changes to the `devel` branch on
+`git.bioconductor.org`. If you have a repository on GitHub, we recommend
+renaming your default branch on GitHub first then renaming your branch in your
+local repository on your computer.
+
+### Rename the Default Branch on Github
```

Comment:
Describe changes needed for local git repositories first

Reply:
I've altered the text to state that if you have a GitHub repository, we recommend changing it there. When you do that, it gives you the information how to change your local repository. But if I understand your comment, you think I write how to do the local changes first?

Reply:
That would be good enough I think. There could also be a section for the few developers (I presume) that only maintain their package(s) locally without a GitHub repo.

Reply:
I am planning to add that. Thanks for your feedback!